default.json

```diff
@@ -95,6 +95,11 @@
       "depTypeList": ["devDependencies"],
       "packageNames": ["@seek/indie-cardib-types"]
     },
+    {
+      "automerge": true,
+      "depTypeList": ["devDependencies"],
+      "packageNames": ["@seek/ie-shared-types"]
```

Comment:
I would just add this to the above array. I don't see a reason to distinguish this from `@seek/indie-cardib-types`

Reply:
Would this auto-merge all renovate PRs? @etaoins / @brian-captain-crosby

Reply:
It will auto-merge all PRs to update `ie-shared-types` once the test goes green. This only works for repos that need a single reviewer; for `indie-graphql-server` is lowers the required reviewer count to 1 instead.

Reply:
Make sense, that's good